helm/chart-operator-chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for the chart-operator
 name: chart-operator-chart
-version: 0.7.1
+version: 0.8.0
```

Comment:
@corest Good to bump the versions here. If you haven't already you also need to bump the release channel in the WIP version of cluster-operator.

https://github.com/giantswarm/cluster-operator/blob/4161b4dfdcb02b73c4574d1c6f9335ac0f514587/pkg/v17/resource/chartoperator/resource.go#L27

Reply:
I'll bump version in cluster-operator later as there is another change expected to arrive here first